kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue

```diff
@@ -1,7 +1,28 @@
 <template>
 
   <div class="full-page">
-    <main class="content">
+    <div
+      v-if="coreError"
+      style="padding: 16px;"
+    >
+      <AppError :hideParagraphs="true">
+        <h2>{{ coreError }}</h2>
+        <template
+          v-if="restart"
```

Comment:
Curious why you hide the "startOver" behind the `restart` prop. I don't think I put this in `OnboardingStepBase`. 

If we only offer the restart on a `400` then that means we won't offer people to restart if they hit a `500` or something, which I think is good because that's not an indication that their request was bad, but that something went wrong server-side.

It just got me wondering if the same logic should be applied throughout or if it just applies to the provisioning case

Reply:
@nucleogenesis This was to purely to target only bad requests. I also share the same thought that 500s should not warrant a restart of a process.  I think it would be good to look at the AppError and assess the feasibility to add it to AppError as a prop. I think other scenarios within Kolibri could benefit from this too.

Reply:
Ahh I see